src/runtime_src/core/common/drv/xrt_cu.c

```diff
@@ -446,6 +459,83 @@ int xrt_cu_cfg_update(struct xrt_cu *xcu, int intr)
 	return err;
 }
 
+/* TODO: maybe we should move below fast adapter specific functions to
+ * fast_adapter.c. Before we clearly know what to do for below cases,
+ *	single CU  to single plram
+ *	multi  CUs to single plram
+ *	single CU  to multi  plrams
```

Comment:
what is not clear here?

Reply:
We discussed this before. Just forgot to delete. Will clean this comment.